pkgs/tools/security/rbw/default.nix

```diff
@@ -22,19 +22,22 @@
   # pass-import
 , withPass ? false
 , pass
+
+  # git-credential-helper
+, withGitCredential ? false
```

Comment:
Why is this off by default? It does not add any new depedencies.

Reply:
i also had that in mind, lastpass-cli just copies that: https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/tools/security/lastpass-cli/default.nix#L41

let me file a new pr.

Reply:
https://github.com/NixOS/nixpkgs/pull/144408/files